### PR TITLE
[MIG][12.0] sale_margin_security: Migration to v12.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT20183
@ernestotejeda @chienandalu Can you review?
@carlosdauden review please

### CODE REVIEW

**Review by ernestotejeda — APPROVED** [review]

File: sale_margin_security/__manifest__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Sale Margin Security",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
```
Comment (carlosdauden) [inline sale_margin_security/__manifest__.py:5]: Add 'development_status': 'Production/Stable',

**Review by carlosdauden — APPROVED** [review]

File: sale_margin_security/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply sale_margin_security/__manifest__.py:5]: Then declare a maintainer as well.

**Comment by sergio-teruel** [comment]
Changes done!!

**Pull request merged**